tenant/config/rbac/rbac_role_binding.yaml

```diff
@@ -10,4 +10,4 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: default
-  namespace: tenant-system
+  namespace: system
```

Comment:
Was this intended?

Reply:
Kubebuilder auto generated code always produces `system` namespace name. I was manually changing it to tenant-system in previous change. I'd rather not do that. The right approach is to use kustomize to add prefix to namespace name. Alternatively, I maintain an all-in-one.yaml to install all components and make namespace adjustment in that file, which is more convenient at this stage.

Reply:
Any reason not to remove it all together so people that don't use kustomize can easily override via `kubectl apply -n tenant-system -f all-in-one.yaml`? I don't really have a preference here, just asking.